Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -2,17 +2,47 @@
 #include "EngineView.h"
 #include "EngineView.g.cpp"
 
+using namespace winrt::Windows::Devices::Input;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::System::Threading;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::UI::Input;
 using namespace winrt::Windows::UI::Xaml;
 using namespace winrt::Windows::UI::Xaml::Input;
 using namespace winrt::Windows::UI::Xaml::Media;
 using namespace winrt::Windows::UI::Xaml::Controls;
 
 namespace winrt::BabylonReactNative::implementation {
+    namespace {
+        constexpr uint32_t LEFT_MOUSE_BUTTON_ID = 0;
+        constexpr uint32_t MIDDLE_MOUSE_BUTTON_ID = 1;
+        constexpr uint32_t RIGHT_MOUSE_BUTTON_ID = 2;
+        constexpr uint32_t TOUCH_BUTTON_ID = 0;

```

Comment:
These must match Babylon.js (which they do), and they must be the same on all platforms, so it seems like it would make more sense for these to be defined (and publicly exposed) in `NativeInput.h`. Otherwise we have to keep them in sync across platforms, right?

Reply:
EngineView doesn't directly reference NativeInput.h currently, so we may need a way to expose these values through BabylonNative.h. Regardless, it sounds good to have these values defined in NativeInput.h somehow

Reply:
i'll take another look at how to propagate this

Reply:
this should now be fixed